ebda_integration/ebda_integration/ebda_api.py

```diff
@@ -35,12 +38,12 @@ def auth(self):
     def check_session(self):
         "params: token | return (status_code, message)"
         url = self.base_url + self.check_session_url
-        params = frappe._dict({"token": self.settings.get_password("token", False)})
+        json = frappe._dict({"token": self.settings.get_password("token", False)})
         try:
-            response = requests.get(url, params=params)
+            response = requests.post(url, json=json, verify=False)
             response.raise_for_status()  # Raises HTTPError for non-200 status codes
             response = response.json()
-            if response.status_code == 200 and response.message == "Token is active":
+            if response.get("status_code") == 200 and response.get("message") == "Token is active":
```

Comment:
Hypothetically according to the standards we should check on the status_code.. the message in this case may not match. if they changed the API.

Reply:
@meaziz the response at documentaion like this:
`{
      "status_code": 200, 
      "message": "Token is Active"
}`